services/app-web/src/components/ChangeHistory/ChangeHistory.tsx

```diff
@@ -29,19 +32,28 @@ export const ChangeHistory = ({
     const { logAccordionEvent } = useTealium()
     const flattenedRevisions = (): flatRevisions[] => {
         const result: flatRevisions[] = []
-
         const contractSubmissions = contract.packageSubmissions.filter(
             (submission) => {
                 return submission.cause === 'CONTRACT_SUBMISSION'
             }
         )
-
+        const reviewActions = contract.reviewStatusActions?.filter(
```

Comment:
this shouldn't be required anymore with the change to the GQL schema, right?

Reply:
yeah holdover from debugging